.github/workflows/publish.yml

```diff
@@ -49,4 +49,11 @@ jobs:
           echo "npmAuthToken: $NODE_AUTH_TOKEN" >> ../../.yarnrc.yml
           yarn workspaces foreach --all --parallel --no-private npm publish --access public
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
\ No newline at end of file
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Trigger Docker Build
+        if: success()
```

Comment:
Isn't this conditional redundant? otherwise we need it also for the 'Publish to npm' step

Reply:
Yea sounds like it is redundant 👍 
https://github.com/env0/env0-backstage-plugin/pull/81/commits/7158621eacff6b06452a72f2440bed95838495a8